plutus-ledger-api/src/Plutus/V1/Ledger/Api.hs

```diff
@@ -120,6 +120,14 @@ internally. That means we don't lose anything by exposing all the details: we're
 anything, we're just going to create new versions.
 -}
 
+{- | Internally the evaluator uses costs which approximate execution times
+in picoseconds.  This gives huge numbers which are unsuitable for users
+so we expose microsecond-based costs to the ledger and scale them up and
```

Comment:
Are we happy with microsecond-level granularity? Or would nanoseconds be better? It seems unlikely to me that any reasonable script is going to be less than a microsecond, but I guess you could care about your script being 1.5micros versus 2micros...

Reply:
Yes, I think we should probably move to nanoseconds, even if the numbers might be a bit big.  Maybe something like tens of nanoseconds or hundreds of nanoseconds (so 10^(-7) or 10^(-8) seconds) would be the right kind of size, but that would be a bit confusing.

Reply:
Okay, let's do nanoseconds then.

Reply:
Done.